src/antsibull/data/ansible-setup_py.j2

```diff
@@ -206,9 +206,14 @@ setup(
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
 {%- endif %}
+{%- if version.major >= 5 and version.major < 7 %}
         'Programming Language :: Python :: 3.8',
+{%- endif %}
```

Comment:
This is OK as a first fix, but we should really make this depend on `python_requires`.

Reply:
Yeah, this'll work for now but it could be improved. We could get ansible-core's classifiers from the PyPI API and copy the ones that start with `Programming Language :: Python :: 3` or something like that. (We could also get rid of they `_python` key in the deps files and retrieve ansible-core's `python_requires` from there.)